### PR TITLE
[marketplace contributions] - fix issue with support labels

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7065)

## Description
* Fixed an issue where support labels were not added specifically in marketplace contributions. See the issues comment for further explanation  
* In case checkout to the contributor's branch fails, fallback to master branch to try to retrieve the support levels of packs.

### CODE REVIEW

**Comment by GuyAfik** [comment]
along with @dansterenson we were able to check that this fix works on marketplace contributions too with this testing PR https://github.com/demisto/content/pull/27617

File: Utils/github_workflow_scripts/handle_external_pr.py [hunk 1]
```diff
@@ -124,10 +124,12 @@ def get_packs_support_level_label(file_paths: List[str], external_pr_branch: str
         f'to retrieve support level of {pack_dirs_to_check_support_levels_labels}'
     )
     try:
+        fork_owner = os.getenv('GITHUB_ACTOR')
         with Checkout(
             repo=Repo(Path().cwd(), search_parent_directories=True),
             branch_to_checkout=external_pr_branch,
-            fork_owner=os.getenv('GITHUB_ACTOR')
+            # marketplace contributions the name of the forked owner is not the same as the name of the owner
```
Comment (ShacharKidor) [inline Utils/github_workflow_scripts/handle_external_pr.py:131]: What is the difference between fork owner and the name of the owner - wording here could be better.

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Nice catch!
Please add a UT for testing MP contributions changes.

**Review by ShacharKidor — APPROVED** [review]

**Comment by GuyAfik** [comment]
working version on the latest commit for marketplace contributions: https://github.com/demisto/content/pull/27657

**Comment by GuyAfik** [comment]
needs force-merge, as the test-upload-flow is broken with validations failing which are un-related to this PR.

**Pull request merged**